README.rst

```diff
@@ -123,17 +124,17 @@ so make sure that it is also available to cmake via the ``CMAKE_PREFIX_PATH``.
 
 For an example setup, take a look at ``examples/cpp/hello_world``.
 
-Documentation
+**Documentation**
 =============
 You can find the API documentation on the pytorch website: https://pytorch.org/docs/stable/torchvision/index.html
 
-Contributing
+**Contributing**
 ============
 We appreciate all contributions. If you are planning to contribute back bug-fixes, please do so without any further discussion. If you plan to contribute new features, utility functions or extensions, please first open an issue and discuss the feature with us.
 
-Disclaimer on Datasets
+**Disclaimer on Datasets**
 ======================
 
-This is a utility library that downloads and prepares public datasets. We do not host or distribute these datasets, vouch for their quality or fairness, or claim that you have license to use the dataset. It is your responsibility to determine whether you have permission to use the dataset under the dataset's license.
+This is a utility library that downloads and prepares the  public datasets. We do not host or redistribute these datasets, vouch for their quality or fairness, or claim that you have license to use the dataset. It is your responsibility to determine whether you have permission to use the dataset under the dataset's license.
```

Comment:
I think the `the` is unnecessary.

The `redistribute` I'm not sure, a Google search s[eemed to look as if `distribute` could be ok](https://wikidiff.com/distribution/redistribution).